Sources/ArcGISToolkit/Components/Search/SearchSource.swift

```diff
@@ -15,7 +15,7 @@
 import ArcGIS
 
 /// Defines the contract for a search result provider.
-public protocol SearchSource {
+public protocol SearchSource: Sendable {
```

Comment:
While adopting the `Sendable` protocol is easy, doing so right can be hard. Neither of the conforming types are actually sendable nor would it be easy to make them so. That is why I didn't do this myself when enabling complete strict concurrency. We may end up having no other choice, but if so, we should make an effort to make the conforming types thread safe, as difficult as that will be. The best option is probably to re-design the API, but I don't think we want to do that right now.

Reply:
I've marked `SearchSource` `@MainActor` and I think it's a suitable solution for now.

Reply:
Since you're adding a new concurrency constraint, make sure to add the `preconcurrency` attribute to the declaration.